Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04 as builder
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+# Timezone data is needed during the installation of dependencies,
+# since one of them depends on tzdata. In an interactive terminal,
```

Comment:
which one?

Reply:
It's in the list of additional packages installed by `cmake`. So the `cmake` package depends on it, either directly or indirectly.